src/apps/rDSN.monitor/rDSN.Monitor.py

```diff
@@ -0,0 +1,37 @@
+﻿import sys
+import os
+import threading
+import time
+sys.path.append(os.getcwd() + '/app_package')
+from MonitorApp import *
+
+def start_dsn():
+    if len(sys.argv) < 2:
+        #rDSN.Monitor run as an embedded service
+        print "rDSN.Monitor runs in embedded mode"
+
+        service_app = ServiceApp()
+        app_dict['monitor'] = MonitorService
+        service_app.register_app('monitor')
+        Native.dsn_app_loader_signal()
+
+        time.sleep(1)
+
+    elif sys.argv[1] == 'standalone':
+        #rDSN.Monitor run as a caller calling the monitored program 
+        print "rDSN.Monitor runs in standalone mode"
+
+        _server_thread= serverThread()
```

Comment:
in standalone mode, you can still register the MonitorService app and configure the ini file to load the http server. What's different is we don't need signal app loader and we need to call dsn_run. In this case, the model is more consistent with the app deployment in rDSN.